pkg/pillar/zedcloud/authen.go

```diff
@@ -514,10 +516,25 @@ func VerifySigningCertChain(log *base.LogObject, certs []*zcert.ZCert) ([]byte,
 			}
 		}
 	}
-	log.Tracef("VerifySigningCertChain: success\n")
 	return sigCertBytes, nil
 }
 
+type certStatus uint8
+
+const (
+	allCertsAreValid certStatus = iota
+	aCertIsInfo
+	aCertIsWarn
+	aCertIsINVALID
+)
+
+var (
+	lastCertTimeCheck time.Time
+	CurCertStatus     = allCertsAreValid
+	certInfoThreshold = 30
+	certWarnThreshold = 15
```

Comment:
Shouldn't these removed and use the variables you added to types/global.go?

Reply:
Just a place holder until I understand and create the appropriate accessors.